composer.json

```diff
@@ -66,8 +67,7 @@
 			"Classes/Testing/ProphecyTrait.php"
 		],
 		"psr-4": {
-			"Sys25\\RnBase\\": "Classes",
-			"Contrib\\": "Resources/Private/Php/Contrib"
```

Comment:
@hannesbochmann du hast die Collections hier entfernt. Das wäre auch mein Plan gewesen. Hast du aber sichergestellt, daß die Doctrine-Collections in einer TER-Installation immer vorhanden sind? Meiner Ansicht nach fehlen die. Somit würde rn_base über TER nicht mehr nutzbar sein.

Reply:
@digedag Hmmm, das weiß ich nicht. Ich bin aber davon ausgegangen, dass auch bei einer TER Installation die gleichen Pakete wie per Composer vorhanden sind.

Reply:
Kein Grund sich zu entschuldigen! Ich nehm die Vars wieder raus und du schaust halt mal, was passiert.